<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="7">

  <testsuite name="appointments"
    errors="0" failures="1" hostname="" id="" package="appointments" skipped="0"
    tests="7" time="103.2" timestamp="Tue, 20 Sep 2016 07:11:59 GMT">
  
    <testcase name="login" time="39.22" assertions="3">

    

    

    
    </testcase>
  
    <testcase name="Set Clinic Hours" time="5.983" assertions="1">

    
    </testcase>
  
    <testcase name="working hours" time="0.002000" assertions="0">
    </testcase>
  
    <testcase name="open patient file" time="14.91" assertions="4">

    

    

    

    
    </testcase>
  
    <testcase name="Doctor Login" time="21.22" assertions="5">

    

    

    

    

    
    </testcase>
  
    <testcase name="Admin Login" time="21.31" assertions="5">

    

    

    

    

    
    </testcase>
  
    <testcase name="Check" time="0.5100" assertions="1">  <failure message="Failed [equal]: (undefined == &#39;5:45 PM&#39;)">Failed [equal]: (undefined == &#39;5:45 PM&#39;): Failed [equal]: (undefined == &#39;5:45 PM&#39;) - Expected &quot;5:45 PM&quot; but got: &quot;undefined&quot;
    at Object.equal (C:\dev1\nightwatch\node_modules\nightwatch\lib\core\api.js:67:24)
    at Object.module.exports.Check (C:\dev1\nightwatch\node_modules\nightwatch\EHRdoctor\appointments.js:382:11)
    at Module.call (C:\dev1\nightwatch\node_modules\nightwatch\lib\runner\module.js:63:34)
    at C:\dev1\nightwatch\node_modules\nightwatch\lib\runner\testcase.js:97:29
    at _fulfilled (C:\dev1\nightwatch\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (C:\dev1\nightwatch\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (C:\dev1\nightwatch\node_modules\q\q.js:796:13)
    at C:\dev1\nightwatch\node_modules\q\q.js:556:49
    at runSingle (C:\dev1\nightwatch\node_modules\q\q.js:137:13)
    at flush (C:\dev1\nightwatch\node_modules\q\q.js:125:13)</failure>

    
    </testcase>
  
  
  </testsuite>
</testsuites>
